lib/delayed/psych_ext.rb

```diff
@@ -74,7 +74,11 @@ def visit_Psych_Nodes_Mapping(object) # rubocop:disable CyclomaticComplexity, Me
             raise Delayed::DeserializationError, "DataMapper::ObjectNotFoundError, class: #{klass} (#{error.message})"
           end
         else
-          super
+          begin
+            super
+          rescue Exception => error
```

Comment:
I recommend `rescue => error` because `Exception` is a little too greedy and catches, for example, `CTRL-C`-ing the process. Leaving out the exception class implies `rescue StandardError => error`.

Reply:
I've changed this to catch `ScriptError` instead of `Exception`.